test/model/message_test.dart

```diff
@@ -375,6 +375,40 @@ void main() {
     });
   });
 
+  group('handleDeleteMessageEvent', () {
+
+    test('delete unknown message', () async {
+      final message1 = eg.streamMessage();
+      final message2 = eg.streamMessage();
+      await prepare();
+      await prepareMessages([message1]);
+      await store.handleEvent(eg.deleteMessageEvent([message2]));
+      checkNotNotified();
+      check(store).messages.values.single.id.equals(message1.id);
+    });
+
+    test('delete messages', () async {
+      final message1 = eg.streamMessage();
+      final message2 = eg.streamMessage();
+      await prepare();
+      await prepareMessages([message1, message2]);
+      await store.handleEvent(eg.deleteMessageEvent([message1, message2]));
+      checkNotifiedOnce();
+      check(store).messages.isEmpty();
+    });
+
+    test('delete an unknown messages with a known messages', () async {
```

Comment:
```suggestion
    test('delete an unknown message and a known message', () async {
```
I think?

Reply:
Oh right, there used to be more messages.